docs/execution.rst

```diff
@@ -4,7 +4,7 @@ Execution
 Queries
 -------
 
-To execute a query against a schema build a new ``GraphQL`` object with the appropriate arguments and then
+To execute a query against a schema build, a new ``GraphQL`` object with the appropriate arguments and then
```

Comment:
small typo: comma should be after 'schema', right?

Reply:
Indeed, good catch! Should be fixed now.